### ISSUE TITLE
Support for pip 6

### ISSUE THREAD

**Issue by sczizzo** [anchor/nagios-plugin-elasticsearch#28]
I see some recent commits around this, but I wanted to ask if there are plans to support pip 6. At present, I receive an error trying to install:

```
$ pip install nagios-plugin-elasticsearch
Collecting nagios-plugin-elasticsearch
  Downloading nagios-plugin-elasticsearch-1.0.2.tar.gz
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-build-g4R6_1/nagios-plugin-elasticsearch/setup.py", line 19, in <module>
        parse_requirements("requirements.txt")],
      File "/usr/local/lib/python2.7/dist-packages/pip/req/req_file.py", line 19, in parse_requirements
        "parse_requirements() missing 1 required keyword argument: "
    TypeError: parse_requirements() missing 1 required keyword argument: 'session'
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

      File "<string>", line 20, in <module>

      File "/tmp/pip-build-g4R6_1/nagios-plugin-elasticsearch/setup.py", line 19, in <module>

        parse_requirements("requirements.txt")],

      File "/usr/local/lib/python2.7/dist-packages/pip/req/req_file.py", line 19, in parse_requirements

        "parse_requirements() missing 1 required keyword argument: "

    TypeError: parse_requirements() missing 1 required keyword argument: 'session'

    ----------------------------------------
    Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-g4R6_1/nagios-plugin-elasticsearch
```

**Comment by hoffoo** [comment]
running into this as well

**Comment by fractalcat** [comment]
Yep, I'll get to this when I can.

**Comment by fractalcat** [comment]
Fix contributed by @zonzamas, merged in #27 and released to pypi. Let me know if there are any issues.
